types/ramda/index.d.ts

```diff
@@ -1598,9 +1598,10 @@ declare namespace R {
         replace(pattern: RegExp | string): (replacement: string) => (str: string) => string;
 
         /**
-         * Returns a new list with the same elements as the original list, just in the reverse order.
+         * Returns a new list or string with the elements or characters in reverse order.
```

Comment:
Would be better to add separate documentation on each overload.

Reply:
Thanks for the suggestion. I have split the documentation in two.